docker-compose.yml

```diff
@@ -196,7 +196,10 @@ services:
         image: ghcr.io/shared-reality-lab/image-handler-svg-semantic-seg:unstable
         labels:
             ca.mcgill.a11y.image.handler: enable
-
+    svg-open-street-map-handler:
+        image: ghcr.io/shared-reality-lab/image-handler-svg-open-street-map:unstable
+        labels:
+            ca.mcgill.a11y.image.handler: enable
```

Comment:
Missing port label. Your container uses port 5000, which is not the default port for HTTP (80).